packages/pn-personagiuridica-webapp/src/utils/constants.ts

```diff
@@ -14,6 +14,9 @@ export const PAGOPA_HELP_EMAIL = process.env.REACT_APP_PAGOPA_HELP_EMAIL;
 // PN-2029
 // export const PAYMENT_DISCLAIMER_URL = process.env.REACT_APP_PAYMENT_DISCLAIMER_URL;
 
+export const LANDING_SITE_URL = process.env.REACT_APP_LANDING_SITE_URL;
+export const FAQ_DIFFERENT_AMOUNTS_SUFFIX = process.env.REACT_APP_FAQ_DIFFERENT_AMOUNTS_SUFFIX;
```

Comment:
@clombardi same comment like the one for persona fisica webapp

Reply:
same as precedent comment